src/readline/readline.c

```diff
@@ -354,616 +353,1773 @@ static int stricmp (), strnicmp();
 /* Non-zero means to save keys that we dispatch on in a kbd macro. */
 static int defining_kbd_macro = 0;
 
-
+
+
+void rl_dispatch (int key, Keymap map);
+void rl_digit_loop ();
+void move_vert (int to);
+void init_terminal_io (char *terminal_name);
+
+
+
 /* **************************************************************** */
 /*								    */
-/*			Top Level Functions			    */
+/*			History Utilities			    */
 /*								    */
 /* **************************************************************** */
 
```

Comment:
what has happened to this file? if nothing was changed, we can revert it?

Reply:
I said I wouldn't take any CR for this file (on Slack group chat) ... :)
I needed to make it compile (it took me a full day). We will remove this anyway.

Reply:
Ok, I feel your pain!